src/components/organisms/PluginManager/PluginManager.tsx

```diff
@@ -42,20 +44,24 @@ const PluginManagerDrawer: React.FC = () => {
     setInstallModalVisible(false);
   };
 
+  const openHelpUrl = () => {
+    const repositoryUrl = `https://github.com/kubeshop/monokle/blob/main/docs/plugins.md`;
```

Comment:
please move this URL to the `src/constants/constants.ts` file as `PLUGINS_HELP_URL`

Reply:
okay

Reply:
done.